RFCs/11-partiql-insert-update-spec.md

```diff
@@ -0,0 +1,728 @@
+- Start Date: 2022-06-17
+- PartiQL Issue [partiql/partiql-docs/#4](https://github.com/partiql/partiql-docs/issues/4)
+- RFC PR: [partiql/partiql-docs/#11](https://github.com/partiql/partiql-docs/pull/11)
+
+# Summary
+[summary]: #summary
+
+This RFC specifies PartiQL's syntax and semantics for insert upsert as part of PartiQL Data Definition Language (DML).
+The RFC outlines the grammar and semantics for PartiQL’s insert or update—also known as UPSERT—and REPLACE statements.
+UPSERT is an informal term for a statement that denotes inserting or updating a set of data to a target database’s table
+based on existing availability condition of that data in the same table.
+
+# Motivation
+[motivation]: #motivation
+
+The PartiQL Specification as it stands today does not specify the syntax and semantics of Data Definition Language (DML).
```

Comment:
Data Manipulation Language?

Reply:
Correct, will fix.

BTW, this PR is still in draft.